README.md

```diff
@@ -4,4 +4,64 @@ SPDX-FileCopyrightText: 2021 Benedict Harcourt <ben.harcourt@harcourtprogramming
 SPDX-License-Identifier: BSD-2-Clause
 -->
 
-# Kitteh's Bot Framework
+# MewBot
+
+MewBot is an automation framework intended to simplify building cross-platform
+text/chat-bots.
+The design is intended to be modular with configuration separated from code,
+allowing users and developers to build out custom logic and behaviours with
+minimal coding experience.
+
+### Status
+
+> :warning: This project is still in the very early stages. Some basic bots can be built
+> and run, but we currently consider all parts of the framework to be unstable.
+
+### Packages and Modules
+
+| Package                   | Modules                             | Description                                                                                                                                |
+|---------------------------|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
No mention of .demo, but presumably this is similar to a scratch so not needed long term.

Reply:
It's more that I don't think that `.demo` will ever need to be published in a package you can download with `pip` and friends.